### ISSUE TITLE
Decks listed as not MWL 3.1 compliant, when they are

### ISSUE THREAD

**Issue by axmccx** [Null-Signal-Games/netrunnerdb#314]
I noticed these two decks are marked as not MWL 3.1 compliant, when as far as I can tell, they are. 

[https://netrunnerdb.com/en/decklist/53996](https://netrunnerdb.com/en/decklist/53996)
[https://netrunnerdb.com/en/decklist/53664](https://netrunnerdb.com/en/decklist/53664)

**Comment by NoahTheDuke** [comment]
Yeah that’s super weird. I’ll have to dig in to see what caused it to fail. Thanks for letting me know

**Comment by metamatik** [comment]
Hi Noah,

Just chiming in with another example in case that helps: https://netrunnerdb.com/en/decklist/54005

One new player wanted to play this deck at our GNK yesterday and got a bit confused as to why NRDB would not qualify it as MWL 3.1 compliant.

I tried to copy it into my decks and edit it.
When I do that I notice that the _Legality_ selected option is _NAPD MWL 2.2 (after 2018-09-06)_.
If I manually select _Standard MWL 3.1 (active)_ and save, NRDB detects no problem.
Hope that helps :)

**Comment by metamatik** [comment]
(side note: I don't really understand the purpose of this select menu to begin with 😅)

**Comment by NoahTheDuke** [comment]
I also have no idea why that menu exists. I suspect excising it is the way to go but I’ll have to check.

I suspect, wrt mwl errors, that for some reason 3.1 errored out, so I might recheck all existing decks for validity just to handle the issue.

**Comment by gereons** [comment]
https://netrunnerdb.com/en/decklist/54005 is an older deck and probably was created when MWL 2.2 was in effect, so that's what the dropdown shows. That's a feature, IMO, as is that copying decks also copies their MWL setting.

I would vote against removing the selection, because it serves both as a display of the currently selected MWL for that deck, but also as a a tool to play with what-if scenarios or even building to older MWL versions for old times' sake.

**Comment by NoahTheDuke** [comment]
Good points! Let's not remove it then.
